Iris/IrisExtension/ExpressionCompiler/DebugCompilerContext.cs

```diff
@@ -46,11 +46,7 @@ internal class DebugCompilerContext : CompilerContext
             List<DkmClrLocalVariableInfo> generatedLocals,
             string assignmentLValue,
             bool argumentsOnly)
-#if NETCOREAPP
-            : this(scope.Session.Importer, reader, CompilationFlags.NoDebug | CompilationFlags.WriteDll | CompilationFlags.NetCore)
```

Comment:
A minor detail... If `CompilationFlags.NetCore` is not set on CompilerContext wouldn't it limit the PEemitter to always look for netframework ilasm on Windows (see [here](https://github.com/microsoft/ConcordExtensibilitySamples/blob/9641ba83e05b0d1f6b7f8c6c645a3c2b0beb9eb4/Iris/IrisCompiler/BackEnd/PeEmitter.cs#L78)) breaking the VSCode-only scenario?

Please correct me, if I am missing something

Reply:
I left `CompilationFlags.NetCore` in as it seemed like a useful concept, but it actually doesn't do much -- [this line](https://github.com/microsoft/ConcordExtensibilitySamples/blob/9641ba83e05b0d1f6b7f8c6c645a3c2b0beb9eb4/Iris/IrisCompiler/BackEnd/PeEmitter.cs#L85) will check if the framework-adjacent ilasm.exe exists, and if not it will go down the 'running on .NET Core' case regardless of the flag.

Reply:
Ah I see, thanks for clarification